src/Libraries/DynamoUnits/UnitsCore.csproj

```diff
@@ -40,7 +40,7 @@
   <Target Name="CopyFiles" BeforeTargets="Build">
     <Copy SourceFiles="$(ProjectDir)DynamoUnits_DynamoCustomization.xml" DestinationFolder="$(OutputPath)" />
     <Copy SourceFiles="$(PkgForgeUnits_NET)\lib\net6.0\ForgeUnitsManaged.dll" DestinationFolder="$(OutputPath)" />
-    <Copy SourceFiles="$(PkgForgeUnits_NET)\runtimes\win10-x64\native\ForgeUnits.dll" DestinationFolder="$(OutputPath)" />
+    <Copy SourceFiles="$(PkgForgeUnits_NET)\runtimes\win10-x64\native\Units.dll" DestinationFolder="$(OutputPath)" />
```

Comment:
does this potentially cause any conflicts with this assembly?

https://github.com/DynamoDS/Dynamo/blob/7c17a2493385799f96139c053273746f181973c8/src/Libraries/UnitsUI/UnitsUI.csproj#L52

Reply:
oh I guess the real assemblyName is DynamoUnits.